### PR TITLE
Adjust column width for main documentation text

### LINKED ISSUE #185: Code snippets overextend layout on narrow browser windows
**Issue description**
Many pages are truncated on the left or right when using a window ~1024px wide.

**Steps to reproduce the issue**
1. Go to https://grpc.io/docs/quickstart/java/
2. Shrink window size

**Desktop (if relevant):**
 - OS: Linux
 - Browser: Chrome

**Additional info**
My browser window tends to stay a bit over 1024 pixels wide, as most content stops being able to use windows over that width.

I just noticed pages where the right-side content was cropped:
https://grpc.io/docs/quickstart/java/
https://grpc.io/docs/guides/auth/

Unless it is the left-side content, because of a fragment:
https://grpc.io/docs/guides/auth/#supported-auth-mechanisms

While very wide-spread, this poor handling doesn't seem universal. Most small pages are fine, and even these:
https://grpc.io/docs/guides/
https://grpc.io/docs/reference/go/generated-code/

I think this is caused by wide code snippets. For example, Ruby, C#, Node.js, Java snippets have long lines on this page:
https://grpc.io/docs/guides/auth/#ruby

Ideally code snippets would scroll instead of causing the page to be truncated. But we also may want to shorten the line length of many of the snippets. Surely we don't need 107 characters on a line.

That may be made worse by the overall large text on the site and generous margin around the toc navigation.

### PR DESCRIPTION
**Pull request by lucperkins**
Fixes #185

### CODE REVIEW

**Comment by chalin** [comment]
In this that there is another root cause to the problem being reported in #185.

**Comment by chalin** [comment]
Looking at the Java QS at 770 px with, this is what I see, which is probably not the result we want:

Screen Shot 2020-04-13 at 17 06 19

**Comment by chalin** [comment]
Or maybe we just need to tweak the breakpoints.

File: layouts/partials/docs/article.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
         {{ partial "nav.html" . }}
       </div>
 
-      <div class="column">
+      <div class="column is-three-quarters">
```
Comment (chalin) [inline layouts/partials/docs/article.html:11]: Don't we want this to be 3/4s only when the left nav is showing?

Comment (lucperkins) [reply layouts/partials/docs/article.html:11]: The column system uses Flexbox to automatically expand and contract these.

**Comment by lucperkins** [comment]
@chalin  I think there's a browser-specific issue at work because this works fine for me on macOS/Firefox. I added a PostCSS pipeline that might help.

**Comment by chalin** [comment]
I see the same problem (the original problem and the issue with the #186 fix) under macOS 10.14.6 and FF 75.0.

**Comment by lucperkins** [comment]
@chalin Unfortunately, eliminating all issues on all browsers and screen sizes is incredibly time intensive. Please feel free to follow up on this work if desired.

**Comment by chalin** [comment]
To be clear: I agree that `is-three-quarters` seemed to be missing, but there is a separate issue with breakpoints mismatching on the left-nav collapse and the adjustment of the column width. But I agree with you that we can address that separately.

**Comment by chalin** [comment]
Here is the followup issue: #188.

File: layouts/partials/docs/article.html (hunk 1 above)
Comment (chalin) [reply layouts/partials/docs/article.html:11]: I can confirm that this change leads to the problem I mentioned at the start of this thread, see #128 

I'll be revert this specific change, reopen the issue and submit a fix.

**Pull request merged**